frontend/src/app/services/resource.service.ts

```diff
@@ -58,4 +59,10 @@ export abstract class ResourceService<T> {
     return throwError(() => error);
   }
 
+  public getFullRequest<TRequest>(url: string, httpParams?: HttpParams): Observable<HttpResponse<TRequest>> {
```

Comment:
Why do you need this type of request?

Reply:
with this method, current class stay more flexible